### PR TITLE
[PMK-8622] Add new items to release notes

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Adds new resume and e-commerce templates to release notes.

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -9,6 +9,49 @@ permalink: /documentation/products/release-notes/
...
+#### Azion Templates
+
+<Tag severity="info" client:only="vue">Preview</Tag>
+
+Azion launched a number of new templates which you can use to deploy an online resume or an e-commerce website on the web.
+
+For **creating an online resume**, you can choose from the following templates:
+
+- **[DevsCard](/en/documentation/products/guides/devscard/)**
+- **[Webpage to PDF Resume](/en/documentation/products/guides/webpage-to-pdf-resume/)**
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx:25]:
```suggestion
- [DevsCard](/en/documentation/products/guides/devscard/)
- [Webpage to PDF Resume](/en/documentation/products/guides/webpage-to-pdf-resume/)
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx:25]: Being a link, it isn't necessary to use the bold style.

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -9,6 +9,50 @@ permalink: /documentacao/produtos/release-notes/
 ---
 import Tag from 'primevue/tag'
 
+## September 30, 2024
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:12]:
```suggestion
## 30 de setembro, 2024
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -9,6 +9,50 @@ permalink: /documentacao/produtos/release-notes/
...
+
+### Marketplace
+
+#### Azion Templates
+
+<Tag severity="info" client:only="vue">Preview</Tag>
+
+A Azion lançou uma série de novos templates que você pode usar para implantar um currículo online ou um site de e-commerce na web.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:20]:
```suggestion
A Azion lançou uma série de novos templates que você pode usar para criar um currículo online ou um site de e-commerce na web.
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 4]
```diff
@@ -9,6 +9,50 @@ permalink: /documentacao/produtos/release-notes/
...
+
+Para **criar um currículo online**, você pode escolher entre os seguintes templates:
+
+- **[DevsCard](/pt-br/documentacao/produtos/guias/devscard/)**
+- **[Webpage to PDF Resume](/pt-br/documentacao/produtos/guias/webpage-to-pdf-resume/)**
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:25]:
```suggestion
- [DevsCard](/pt-br/documentacao/produtos/guias/devscard/)
- [Webpage to PDF Resume](/pt-br/documentacao/produtos/guias/webpage-to-pdf-resume/)
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**